javav2/example_code/cloudwatch/Readme.md

```diff
@@ -1,90 +1,98 @@
+- [Get a metric data image](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java)  (getMetricWidgetImage command)
+- [List metrics](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java)  (ListMetrics command)
+- [Put data into a metric](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java)  (putMetricData command)
 
- ## Testing the Amazon CloudWatch Java files
 
-You can test the Java code examples for Amazon CloudWatch by running a test file named **CloudWatchTest**. This file uses JUnit 5 to run the JUnit tests and is located in the **src/test/java** folder. For more information, see [https://junit.org/junit5/](https://junit.org/junit5/).
+### Scenario 
 
-You can run the JUnit tests from a Java IDE, such as IntelliJ, or from the command line by using Maven. As each test is run, you can view messages that inform you if the various tests succeed or fail. For example, the following message informs you that Test 3 passed.
+Code examples that show you how to accomplish a specific task by calling multiple functions within the same service.
 
-	Test 3 passed
+- [Get started with billing, alarms, and metrics](https://github.com/awsdocs/aws-doc-sdk-examples/blob/main/javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java)  (multiple commands)
 
-**WARNING**: _Running these JUnit tests manipulates real Amazon CloudWatch resources and may incur charges on your account._
+## Run the examples
 
- ### Properties file
-Before running the Amazon CloudWatch JUnit tests, you must define values in the **config.properties** file located in the **resources** folder. This file contains values that are required to run the JUnit tests. For example, you define an alarm name for various tests. If you do not define all values, the JUnit tests fail.
+### Prerequisites
 
-Define these values to successfully run the JUnit tests:
+To run these examples, set up your development environment. For more information, 
+see [Get started with the SDK for Java](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/setup.html). 
 
-- **logGroup** - The name of the log group to use. For example, **testgroup**.
+To run these examples, you must have the following three JSON files: jsonWidgets.json, CloudDashboard.json, and settings.json. Find these files in this GitHub repository. The CloudWatch scenario depends on these files.
+ 
+ **Be very careful** when running an operation that deletes or modifies AWS resources in your account. We recommend creating separate test-only resources when experimenting with these examples.
```

Comment:
You could also add a note about enabling billing alarms to get billing data? The .NET readme has an example with a link.

Reply:
Nice suggestion.